### ISSUE TITLE
ggemmeans's backtransformation to probabilities from MixMod GLMMadaptive (vs emmeans)

### ISSUE THREAD

**Issue by mamscience** [strengejacke/ggeffects#269]
Hi Daniel

Thanks for authoring and maintaining this excellent package. 

I've built a GLMMadaptive MixMod with _family= binomial(link="logit" )_, and the model appears to be performing well.

Now, when I use ggemmeans, it returns predicted probabilities, which is great.  However, when I want contrasts between groups and use emmeans, it doesn't want to return the response scale but rather returns on the logit scale (I think).  Fiddling around with emmeans actually made me notice that it ignores "type=response" and in all cases returns the estimates on the logit scale, not the response scale. I'm even uncertain MixMod is supported by emmeans.

Now, I was wondering how ggemmeans gets predicted probabilities from a MixMod (GLMMadaptive) model. I've searched the source code but couldn't find anything particular other than a custom backtransformation function. Could you point to how could apply this backtransformation, so that I can seek to apply it myself with the output of emmeans. 

Hope you can help, cheers. 

Example Model taken from: https://drizopoulos.github.io/GLMMadaptive/articles/Multiple_Comparisons.html
```
set.seed(1234)
n <- 300 # number of subjects
K <- 4 # number of measurements per subject
t_max <- 15 # maximum follow-up time

# we construct a data frame with the design: 
# everyone has a baseline measurement, and then measurements at K time points
DF <- data.frame(id = rep(seq_len(n), each = K),
                 time = gl(K, 1, n*K, labels = paste0("Time", 1:K)),
                 sex = rep(gl(2, n/2, labels = c("male", "female")), each = K))

# design matrices for the fixed and random effects
X <- model.matrix(~ sex * time, data = DF)
Z <- model.matrix(~ 1, data = DF)

betas <- c(-2.13, 1, rep(c(1.2, -1.2), K-1)) # fixed effects coefficients
D11 <- 1 # variance of random intercepts

# we simulate random effects
b <- cbind(rnorm(n, sd = sqrt(D11)))
# linear predictor
eta_y <- as.vector(X %*% betas + rowSums(Z * b[DF$id, ]))
# we simulate binary longitudinal data
DF$y <- rbinom(n * K, 1, plogis(eta_y))

gm <- mixed_model(fixed = y ~ sex * time, random = ~ 1 | id, data = DF,
                  family = binomial())
```

Compare
```
ggemmeans(gm, c("sex","time"))
emmeans(gm, ~ sex | time, type="response")
emmeans(gm ,specs =pairwise ~ sex | time,  type="response")
```

**Comment by mamscience** [comment]
My apologies, I see how easy it is to calculate back from logit to probabilities using the inverse function. (many examples, one being inv.logit(model).

I'm struggling with the fact that when you contrasts with estimates on the logistic scale, small differences in logits will lead to probabilities close to 0.5. But that's not a ggemmeans-issue, of course

**Comment by strengejacke** [comment]
You can backtransform using [`insight::link_inverse()`](https://easystats.github.io/insight/reference/link_inverse.html), for many models. If you need contrasts, you could also try the [_marginaleffects](https://vincentarelbundock.github.io/marginaleffects/) package, which is easy to use, supports lots of models and has a great documentation. Tagging @vincentarelbundock in case he'd like to add something to this discussion.

**Comment by mamscience** [comment]
Thanks Daniel, afaik MixMod is not supported by marginaleffects... the package actually refers to ggemmeans ;-)
Issue is solved here btw: https://github.com/rvlenth/emmeans/issues/378

**Comment by strengejacke** [comment]
Thanks! So I can close this, right?
